deps/devdoc/requirement_docs/com/microsoft/azure/sdk/iot/deps/serializer/authenticationparser_requirements.md

```diff
@@ -1,73 +0,0 @@
-# AuthenticationParser Requirements
-
```

Comment:
Were these hand written or auto-generated?

Reply:
Hand written, and long since forgotten about